test/pom.xml

```
@@ -270,6 +270,21 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>**/hadoop-metrics2-accumulo.properties</exclude>
+          </excludes>
+          <archive>
+            <manifestEntries>
+              <Sealed>true</Sealed>
+              <Implementation-Build>${mvngit.commit.id}</Implementation-Build>
+            </manifestEntries>
+          </archive>
```

This can be omitted, as it would be merged with the configuration from the parent anyway. We only need to specify the portions that we wish to override in here.
```suggestion
          </excludes>
```